client/package.json

```diff
@@ -7,7 +7,7 @@
     "test": "jest",
     "ci": "npm run prettier && npm run lint && npm test",
     "lint": "eslint src",
-    "prettier": "prettier --write 'src/**/*.js'"
+    "prettier": "prettier --write ."
```

Comment:
Change this in server/package.json as well?

Reply:
Actually I will fix it in another PR (along with the "npm run start" script)